libp2p.nimble

```diff
@@ -92,17 +90,11 @@ task website, "Build the website":
   exec "mkdocs build"
 
 task examples_build, "Build the samples":
```

Comment:
nit: let's rename task to `examples`
`task examples, "Build and run examples":

because it 'conflicts' with `examples_build` below